components/Ethdriver/include/plat/imx8mm-evk/plat/eth_devices.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019, Data61
+ * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+ * ABN 41 687 119 230.
+ *
+ * This software may be distributed and modified according to the terms of
+ * the GNU General Public License version 2. Note that NO WARRANTY is provided.
+ * See "LICENSE_GPLv2.txt" for details.
+ *
+ * @TAG(DATA61_GPL)
```

Comment:
this is no longer used, change to 
```suggestion
 *  * SPDX-License-Identifier: GPL-2.0-only
```
I wonder, should this become GPL or BSD?

Reply:
You're right, by default it should be BSD unless there is some specific reason for GPL (i.e. a GPL dependency or otherwise derived from GPL code).

The rest of the copyright header should also follow the new shorter line, and unless this code was written at CSIRO already (more than a year ago), it should probably be copyright UNSW.

Reply:
This component is GPL due to licensing requirements inherited from using GPL ethernet drivers from U-Boot.

Reply:
So this contains code that is copied from U-Boot? In that case, the original U-Boot copyright headers should be here and/or a comment where exactly the code is from. Ot is this linked together with some U-Boot code?

Reply:
The header file doesn't contain code from U-Boot, but the Ethdriver component does which makes the whole component a derived work.

Reply:
I still don't see any mention of U-Boot anywhere in the repo apart from TimeServer. Am I missing something?